js/bitmex.js

```diff
@@ -221,44 +223,179 @@ module.exports = class bitmex extends Exchange {
                 'fetchOHLCVOpenTimestamp': true,
                 'networks': {
                     'BTC': 'btc',
-                    'ETH': 'eth',
-                    'BSC': 'bsc',
-                    'BNB': 'bsc',
-                    'TRON': 'tron',
+                    'BITCOIN': 'btc',
                     'ERC20': 'eth',
-                    'BEP20': 'bsc',
+                    'ETH': 'eth',
                     'TRC20': 'tron',
                     'TRX': 'tron',
-                    'AVAX': 'avax',
+                    'BEP20': 'bsc',
+                    'SOLANA': 'sol',
+                    'AVALANCHEC': 'avax',
                     'NEAR': 'near',
-                    'XTZ': 'xtz',
-                    'DOT': 'dot',
-                    'SOL': 'sol',
+                    'TEZOS': 'xtz',
+                    'POLKADOT': 'dot',
+                    'CARDANO': 'ada',
                 },
                 'networksById': {
                     'btc': 'BTC',
                     'eth': 'ERC20',
-                    'bsc': 'BSC',
-                    'tron': 'TRX',
-                    'avax': 'AVAX',
+                    'tron': 'TRC20',
+                    'bsc': 'BEP20',
+                    'sol': 'SOLANA',
+                    'avax': 'AVALANCHEC',
                     'near': 'NEAR',
-                    'xtz': 'XTZ',
-                    'dot': 'DOT',
-                    'sol': 'SOL',
+                    'xtz': 'TEZOS',
+                    'dot': 'POLKADOT',
+                    'ada': 'CARDANO',
                 },
             },
             'commonCurrencies': {
-                'USDt': 'USDT',
-                'XBt': 'BTC',
                 'XBT': 'BTC',
-                'Gwei': 'ETH',
                 'GWEI': 'ETH',
                 'LAMP': 'SOL',
                 'LAMp': 'SOL',
             },
         });
     }
 
+    async fetchCurrencies (params = {}) {
```

Comment:
@ttodua I think we might have a problem here, for instance bitmex has doge markets but `DOGE` is not a currency 
![image](https://user-images.githubusercontent.com/43336371/211791726-2ecd54ce-bcca-4a91-9708-70b8495d88a0.png)

Reply:
fixed.